board/safety/safety_hyundai_canfd.h

```diff
@@ -334,6 +335,13 @@ static int hyundai_canfd_tx_hook(CANPacket_t *to_send) {
     }
   }
 
+  // ADRV_0x160 (for FCA warning light), only allowed if openpilot longitudinal
+  if (addr == 0x160) {
+    if (!hyundai_longitudinal) {
+      tx = 0;
+    }
+  }
+
```

Comment:
We don't know if this actuates AEB, @sunnyhaibin can you make a new PR for https://github.com/commaai/openpilot/pull/30021 to add some simple safety around the signals (assert static values from hyundaicanfd.py)?